lib/mix/task_certs_dev.ex

```diff
@@ -0,0 +1,56 @@
+defmodule Mix.Tasks.Certs.Dev do
+  use Mix.Task
+
+  @shortdoc "Generate fake certs (placeholders) for HTTPS endpoint and APNS"
+  @moduledoc """
+  Generate fake certs (placeholders) for `HTTPS` endpoint and `APNS` service.
+
+  Please be aware that `APNS` requires valid Apple Developer certificates, so it
+  will not accept those fake certificates. Generated certificates may be used
+  only with mock APNS service (like one provided by docker
+  `mobify/apns-http2-mock-server`).
+  """
+
+  @spec run(term) :: :ok
+  def run(_) do
+    maybe_gen_dev_apns()
+    maybe_gen_prod_apns()
+    maybe_gen_https()
+  end
+
+  defp maybe_gen_dev_apns do
+    maybe_gen_cert("priv/apns/dev_cert.pem", "priv/apns/dev_key.pem",
+                   "mongoose-push-apns-dev")
+  end
+
+  defp maybe_gen_prod_apns do
+    maybe_gen_cert("priv/apns/prod_cert.pem", "priv/apns/prod_key.pem",
+                   "mongoose-push-apns-prod")
+  end
+
+  defp maybe_gen_https do
+    maybe_gen_cert("priv/ssl/fake_cert.pem", "priv/ssl/fake_key.pem",
+                   "mongoose-push")
+  end
+
+  defp maybe_gen_cert(cert_file, key_file, common_name) do
+    unless File.exists?(cert_file) and File.exists?(key_file) do
```

Comment:
Maybe you can simply write `if File.exists? ... and ... do ok else do_stuff() end`